job-scheduler/src/test/java/org/hawkular/metrics/scheduler/impl/JobSchedulerTest.java

```diff
@@ -112,8 +112,8 @@ protected static void setActiveTimeSlices(DateTime... timeSlices) {
         }
     }
 
-    protected static Set<JobDetails> getScheduledJobs(DateTime timeSlice) {
-        List<JobDetails> jobs = jobsService.findScheduledJobsForTime(timeSlice.toDate(), Schedulers.computation())
+    protected static Set<JobDetailsImpl> getScheduledJobs(DateTime timeSlice) {
```

Comment:
I would use the interface here too since you do not need access to any specific impl details.